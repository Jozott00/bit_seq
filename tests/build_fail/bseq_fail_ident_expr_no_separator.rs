use bit_seq::bseq;

fn main() {
    let var = 0;
    bseq!(var 2);
}