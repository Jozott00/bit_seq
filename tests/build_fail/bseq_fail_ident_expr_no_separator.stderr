error: expected `:`
 --> tests/build_fail/bseq_fail_ident_expr_no_separator.rs:5:15
  |
5 |     bseq!(var 2);
  |               ^
